### ISSUE TITLE
HW 7: Question 3

### ISSUE THREAD

**Issue by dougmantis** [scoskey/m301#4]
> Can you construct a $2\times 2$ matrix $A$ such that the sum of the rows of $A$ is the vector $(1,1)$ and the sum of the columns of $A$ is the vector $(0,0)$? If not, why not?

This question comes in two parts: 

1. Can you construct a matrix such that the sum of the rows is the vector (1,1) and the sum of the columns is the vector (0,0)?
2. If not, why not?

The first part of this problem can be answered by assumption. If the answer were to be 'yes', the student could just say 'yes' as the answer to the whole problem. Since part two of the problem hinges on being 'no', and there is no part two to the branch of the answer being 'yes', students can assume that the answer is no, because otherwise there would be a contingency for it.

**TLDR**: It's easy to tell that the answer is no, because otherwise you would've asked for an example of the kind of vector you're looking for.

**Possible solution:**

Add: "If so, give an example of such a vector.", so students have to go through the process of finding out if you can or can't, without assuming it's 'no' from the get-go.
